.env.default

```diff
@@ -18,3 +18,8 @@ MARKETPLACE_FAVORITES_SERVER_URL=https://marketplace-favorites-api.decentraland.
 WKC_METRICS_RESET_AT_NIGHT=false
 WERT_PRIVATE_KEY='dummyKey'
 WERT_PUBLICATION_FEES_PRIVATE_KEY='dummyKey2'
+SEGMENT_WRITE_KEY='dummyKey3'
+BUILDER_SERVER_DB_USER='dummyUser'
+BUILDER_SERVER_DB_PASSWORD='dummyPassword'
+PG_COMPONENT_PSQL_HOST='dummyHost'
+PG_COMPONENT_PSQL_PORT=5432
```

Comment:
should be defined here the env var: `PG_COMPONENT_PSQL_CONNECTION_STRING`?

Reply:
yeah, it could added in here, added!